assets/translations/en.json

```
@@ -168,8 +168,9 @@
     "enter_email_mess": "Please enter your email address",
     "invalid_email": "Invalid email address",
     "email_address": "Email address",
-    "email_sent": "Reset email has been sent",
-    "email_send_error": "That email does not exist in the system.",
+    "email_sent_1": "Reset email has been sent to",
+    "email_sent_2": "if the account exists.",
```

This is fine, but just some food for thought:

Ideally this would just be 1 translation key using token replacement, which makes translation keys less confusing to translators:
```suggestion
    "email_send_success": "Reset email has been sent to\n<1>{{email}}</1>\nif the account exists.",
```

Then you'd have something like this in your code:

```tsx
        <Text fontSize={14}>
          <Trans i18nKey="forgot_pass_form:email_send_success">
            Reset email has been sent to
            <Text fontSize={14} fontWeight="800">
              <>{{ email: form.getValues().email }}</>
            </Text>
            if the account exists.
          </Trans>
        </Text>
```

I haven't tested it, there might be some minor corrections needed above.

This technique for replacing tokens inside rich text is often used in the admin panel (see for examples).

---

It works this way:

```jsx
// Component
<Text>
  <Trans
    i18nKey="forgot_pass_form:email_send_success"
    components={{
      strong: <Text fontSize={14} fontWeight="800" />,
    }}
    values={{ email: props.email }}
  />
</Text>
```

```js
// Translation
{
  "email_send_success": "Reset email has been sent to <strong>{{email}}</strong> if the account exists.",
}
```

just amazing !! thanks @farmerpaul